### PR TITLE
BR: only update 5.4 parts of BR FAQ

### PR DESCRIPTION
**Pull request by en-jin19**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/7515
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/backup-and-restore-faq.md [hunk 1]
```diff
@@ -8,6 +8,23 @@ aliases: ['/docs/dev/br/backup-and-restore-faq/']
 
 This document lists the frequently asked questions (FAQs) and the solutions about Backup & Restore (BR).
 
+If you encountered problems that are not listed in this document and cannot be resolved, you are welcome to ask your questions to [AskTUG](https://asktug.com/) community.
+
+## In TiDB v5.4.0 and later versions, when backup tasks perform on the cluster under high workload, why does the speed of backup tasks become slow?
+
+Starting from TiDB v5.4.0, TiKV introduces the auto-tune feature for backup tasks. For clusters in v5.4.0 or later versions, this feature is enabled by default. When the cluster workload is high, the feature limits the resources used by backup tasks to reduce the impact on the online cluster. For more information, refer to [BR Auto-Tune](/br/br-auto-tune.md).
+
+TiKV supports [dynamically configuring](tikv-control.md#modify-the-tikv-configuration-dynamically) the auto-tune feature. You can enable or disable the feature by the following mothods without restarting your cluster:
```
Comment (ran-huang) [inline br/backup-and-restore-faq.md:17]:
```suggestion
TiKV supports [dynamically configuring](tikv-control.md#modify-the-tikv-configuration-dynamically) the auto-tune feature. You can enable or disable the feature by the following methods without restarting your cluster:
```

File: br/backup-and-restore-faq.md [hunk 2]
```diff
@@ -8,6 +8,23 @@ aliases: ['/docs/dev/br/backup-and-restore-faq/']
...
+
+- Disanable auto-tune: Set the TiKV configuration item [`backup.enable-auto-tune`](/tikv-configuration-file.md#enable-auto-tune-new-in-v540) to `false`.
```
Comment (ran-huang) [inline br/backup-and-restore-faq.md:19]:
```suggestion
- Disable auto-tune: Set the TiKV configuration item [`backup.enable-auto-tune`](/tikv-configuration-file.md#enable-auto-tune-new-in-v540) to `false`.
```

File: br/backup-and-restore-faq.md [hunk 3]
```diff
@@ -8,6 +8,23 @@ aliases: ['/docs/dev/br/backup-and-restore-faq/']
...
+- Enable auto-tune: Set `backup.enable-auto-tune` to `true`. For clusters that upgrade from versions earlier than v5.4.0 to v5.4.0 or later versions, the auto-tune feature is disabled by default. You need to manually enable it.
```
Comment (ran-huang) [inline br/backup-and-restore-faq.md:20]:
```suggestion
- Enable auto-tune: Set `backup.enable-auto-tune` to `true`. For clusters that upgrade from v5.3.x to v5.4.0 or later versions, the auto-tune feature is disabled by default. You need to manually enable it.
```

File: br/backup-and-restore-faq.md [hunk 4]
```diff
@@ -8,6 +8,23 @@ aliases: ['/docs/dev/br/backup-and-restore-faq/']
...
+
+For the information on modifying command lines of auto-tune online, refer to [Use auto-tune](/br/br-auto-tune.md#use-auto-tune).
```
Comment (ran-huang) [inline br/backup-and-restore-faq.md:22]:
```suggestion
To use `tikv-ctl` to enable or disable auto-tune, refer to [Use auto-tune](/br/br-auto-tune.md#use-auto-tune).
```

File: br/backup-and-restore-faq.md [hunk 5]
```diff
@@ -8,6 +8,23 @@ aliases: ['/docs/dev/br/backup-and-restore-faq/']
...
+
+In addition, this feature also reduces the default number of threads used by backup tasks when performing backup tasks. For details, see `backup.num-threads`](/tikv-configuration-file.md#num-threads-1). Therefore, the speed, CPU usage, and I/O resource utilization used by backup tasks which you can see on the Grafana Dashboard are lower than those of versions earlier than v5.4. Before v5.4, the default value of `backup.num-threads` was CPU * 0.75, that is, the number of threads used by backup tasks makes up 75% of the logical CPU cores. The maximum value of it was `32`. Starting from v5.4, the default value of this configuration item is CPU * 0.5, and its maximum value is `8`.
```
Comment (ran-huang) [inline br/backup-and-restore-faq.md:24]:
```suggestion
In addition, this feature also reduces the default number of threads used by backup tasks. For details, see `backup.num-threads`](/tikv-configuration-file.md#num-threads-1). Therefore, on the Grafana Dashboard, the speed, CPU usage, and I/O resource utilization used by backup tasks are lower than those of versions earlier than v5.4. Before v5.4, the default value of `backup.num-threads` was `CPU * 0.75`, that is, the number of threads used by backup tasks makes up 75% of the logical CPU cores. The maximum value of it was `32`. Starting from v5.4, the default value of this configuration item is `CPU * 0.5`, and its maximum value is `8`.
```

File: br/backup-and-restore-faq.md [hunk 6]
```diff
@@ -16,9 +33,37 @@ It is recommended to mount an NFS disk as a backup disk during backup. For detai
 
 ## How much does it affect the cluster during backup using BR?
 
-When you use the `oltp_read_only` scenario of `sysbench` to back up to a disk (make sure the backup disk and the service disk are different) at full rate, the cluster QPS is decreased by 15%-25%. The impact on the cluster depends on the table schema.
+- For TiDB v5.4.0 or later versions:
+
+    BR not only reduces the default CPU utilization used by backup tasks but also introduces auto-tune feature. After this feature is enabled, BR can automatically limit the resources used by backup tasks when performing backups in the cluster with high workloads, thereby limiting the speed of BR backups. Therefore, when using the default configuration for backup tasks in the cluster in v5.4.0 with high workloads, the impact of the tasks on the cluster performance is significantly less than the impact for the clusters earlier than v5.4.0. For details on the auto-tune feature, see [BR Auto-tune](/br/br-auto-tune.md).
```
Comment (ran-huang) [inline br/backup-and-restore-faq.md:38]:
```suggestion
    BR not only reduces the default CPU utilization used by backup tasks but also introduces the auto-tune feature. When this feature is enabled, BR automatically limits the resources used by backup tasks when performing backups in the cluster with high workloads, thereby limiting the speed of BR backups. Therefore, when using the default configuration for backup tasks in the cluster in v5.4.0 with high workloads, the impact of the tasks on the cluster performance is significantly less than the impact for the clusters earlier than v5.4.0. For details on the auto-tune feature, see [BR Auto-tune](/br/br-auto-tune.md).
```

File: br/backup-and-restore-faq.md [hunk 7]
```diff
@@ -16,9 +33,37 @@ It is recommended to mount an NFS disk as a backup disk during backup. For detai
...
+
+    The following is an internal test on a single node. The test result shows that when using the default configuration of v5.4.0 and its earlier versions in the **full-speed backup** scenario, the impact of backup using BR on cluster performance is quite different. The detailed test results are as follows:
```
Comment (ran-huang) [inline br/backup-and-restore-faq.md:40]:
```suggestion
    The following is an internal test on a single node. The test results show that when using the default configuration of v5.4.0 and its earlier versions in the **full-speed backup** scenario, the impact of backup using BR on cluster performance is quite different. The detailed test results are as follows:
```

File: br/backup-and-restore-faq.md [hunk 8]
```diff
@@ -16,9 +33,37 @@ It is recommended to mount an NFS disk as a backup disk during backup. For detai
...
+
+    - When using the default configuration of v5.3.0, the QPS of pure write load is reduced by 75%.
```
Comment (ran-huang) [inline br/backup-and-restore-faq.md:42]:
```suggestion
    - When BR uses the default configuration of v5.3.0, the QPS of write-only workload is reduced by 75%.
```

File: br/backup-and-restore-faq.md [hunk 9]
```diff
@@ -16,9 +33,37 @@ It is recommended to mount an NFS disk as a backup disk during backup. For detai
...
+    - When using the default configuration of v5.4.0, the QPS for the same load is reduced by 25%. However, when this configuration is used, the speed of backup tasks using BR becomes correspondingly slower. The time required is 1.7 times that of the v5.3.0 configuration.
```
Comment (ran-huang) [inline br/backup-and-restore-faq.md:43]:
```suggestion
    - When BR uses the default configuration of v5.4.0, the QPS for the same workload is reduced by 25%. However, when this configuration is used, the speed of backup tasks using BR becomes correspondingly slower. The time required is 1.7 times that of the v5.3.0 configuration.
```

File: br/backup-and-restore-faq.md [hunk 10]
```diff
@@ -16,9 +33,37 @@ It is recommended to mount an NFS disk as a backup disk during backup. For detai
...
+
+- For TiDB v5.3.0 or earlier versions:
+
+    In these versions, the default configuration parameters of BR only serve to **offline backup** tasks. In the default configuration, the backup tasks using BR might use a large amount of CPU and I/O resources of the cluster, which caused the cluster latency to increase.
```
Comment (ran-huang) [inline br/backup-and-restore-faq.md:47]:
```suggestion
    In these versions, the default configuration parameters of BR only serve to **offline backup** tasks. In the default configuration, the backup tasks using BR might use a large amount of CPU and I/O resources of the cluster, which causes the cluster latency to increase.
```

File: br/backup-and-restore-faq.md [hunk 11]
```diff
@@ -16,9 +33,37 @@ It is recommended to mount an NFS disk as a backup disk during backup. For detai
...
+
+    The results of internal tests show that under the default configuration, the backup tasks using BR have a great impact on the cluster performance. The detailed test information is as follows:
+
+    - Test environment: Use a TiKV cluster with 6 nodes and a single-table dataset to perform the following hybrid operations, whose ratio of the read and write tasks is about `10:1`.
```
Comment (ran-huang) [inline br/backup-and-restore-faq.md:51]:
```suggestion
    - Test environment: Use a cluster with 6 TiKV nodes and a single-table dataset to perform the following hybrid operations. The ratio of the read and write tasks is about `10:1`.
```

File: br/backup-and-restore-faq.md [hunk 12]
```diff
@@ -16,9 +33,37 @@ It is recommended to mount an NFS disk as a backup disk during backup. For detai
...
+    - Test mothods:
```
Comment (ran-huang) [inline br/backup-and-restore-faq.md:52]:
```suggestion
    - Test methods:
```

File: br/backup-and-restore-faq.md [hunk 13]
```diff
@@ -16,9 +33,37 @@ It is recommended to mount an NFS disk as a backup disk during backup. For detai
...
+        - Insert to a single node
```
Comment (ran-huang) [inline br/backup-and-restore-faq.md:53]: 单点插入 指的是 row 不是 node 吧？
@YuJuncen 这里怎么翻译比较好呢？

File: br/backup-and-restore-faq.md [hunk 14]
```diff
@@ -16,9 +33,37 @@ It is recommended to mount an NFS disk as a backup disk during backup. For detai
...
+        - Update a single
```
Comment (ran-huang) [inline br/backup-and-restore-faq.md:54]: ditto

File: br/backup-and-restore-faq.md [hunk 15]
```diff
@@ -16,9 +33,37 @@ It is recommended to mount an NFS disk as a backup disk during backup. For detai
...
+
+    The following is an internal test on a single node. The test result shows that when using the default configuration of v5.4.0 and its earlier versions in the **full-speed backup** scenario, the impact of backup using BR on cluster performance is quite different. The detailed test results are as follows:
+
+    - When using the default configuration of v5.3.0, the QPS of pure write load is reduced by 75%.
+    - When using the default configuration of v5.4.0, the QPS for the same load is reduced by 25%. However, when this configuration is used, the speed of backup tasks using BR becomes correspondingly slower. The time required is 1.7 times that of the v5.3.0 configuration.
+
+- For TiDB v5.3.0 or earlier versions:
```
Comment (ran-huang) [inline br/backup-and-restore-faq.md:45]: 1. 我记得当时讨论过说[移除 5.3 的部分](https://github.com/pingcap/docs-cn/pull/7515#discussion_r781734663)？
2. 如果不能移除，能否直接给结论，不要描述测试过程了，有点冗长。

File: br/backup-and-restore-faq.md [hunk 16]
```diff
@@ -16,9 +33,37 @@ It is recommended to mount an NFS disk as a backup disk during backup. For detai
...
+        - Update a single
+        - Queries for small ranges
+    - Benchmark: Total QPS is around 20K, P95 latency is around 28ms, P99 is around 50ms.
+    - Test result: When the total CPU utilization of the cluster is controlled at 91% by limiting speed, the results are as follows:
+        - The QPS of **write load** reduced by 50%, and the P99 latency increased several times.
+        - In contrast, **read load** is less affected, and its QPS even increased slightly.
+        - The overall P99 latency only rose by around 50%, and QPS reduced by around 4%.
+
+
+If you need to manually control the impact of backup tasks on cluster performance, you can use the following solutions. These two methods can not only reduce the impact of backup tasks on the cluster but also reduce the speed of backup tasks.
```
Comment (ran-huang) [inline br/backup-and-restore-faq.md:63]:
```suggestion
If you need to manually control the impact of backup tasks on cluster performance, you can use the following solutions. These two methods can reduce the impact of backup tasks on the cluster, but they also reduce the speed of backup tasks.
```
原翻译的逻辑关系有点问题

File: br/backup-and-restore-faq.md [hunk 17]
```diff
@@ -8,6 +8,23 @@ aliases: ['/docs/dev/br/backup-and-restore-faq/']
 
 This document lists the frequently asked questions (FAQs) and the solutions about Backup & Restore (BR).
 
+If you encountered problems that are not listed in this document and cannot be resolved, you are welcome to ask your questions to [AskTUG](https://asktug.com/) community.
+
+## In TiDB v5.4.0 and later versions, when backup tasks perform on the cluster under high workload, why does the speed of backup tasks become slow?
+
+Starting from TiDB v5.4.0, TiKV introduces the auto-tune feature for backup tasks. For clusters in v5.4.0 or later versions, this feature is enabled by default. When the cluster workload is high, the feature limits the resources used by backup tasks to reduce the impact on the online cluster. For more information, refer to [BR Auto-Tune](/br/br-auto-tune.md).
```
Comment (YuJuncen) [inline br/backup-and-restore-faq.md:15]: IMHO, "heavy workload" sounds better.
image

(Seems there are other "high workloads" in the document, ditto if possible.)

```suggestion
Starting from TiDB v5.4.0, TiKV introduces the auto-tune feature for backup tasks. For clusters in v5.4.0 or later versions, this feature is enabled by default. When the cluster workload is heavy, the feature limits the resources used by backup tasks to reduce the impact on the online cluster. For more information, refer to [BR Auto-Tune](/br/br-auto-tune.md).
```

File: br/backup-and-restore-faq.md (hunk 13 above)
Comment (YuJuncen) [reply br/backup-and-restore-faq.md:53]: How about...

- Point insert.
- Point update.

File: br/backup-and-restore-faq.md (hunk 15 above)
Comment (en-jin19) [reply br/backup-and-restore-faq.md:45]: 1. 这个是会放到 master 和 5.4 的，所以会保留 5.3 的部分。等重新在 5.3 分支下提 PR 时，会移除上半部分的 5.4 的部分，并把新 PR 中 cp 到 4.0~ 5.3 的版本。
2. 之前记得也可以保留测试流程，因为是列为无序列表，看起来比较多，但内容其实并不多

File: br/backup-and-restore-faq.md [hunk 18]
```diff
@@ -8,6 +8,22 @@ aliases: ['/docs/dev/br/backup-and-restore-faq/']
 
 This document lists the frequently asked questions (FAQs) and the solutions about Backup & Restore (BR).
 
+
+## In TiDB v5.4.0 and later versions, when backup tasks are performed on the cluster under high workload, why does the speed of backup tasks become slow?
+
+Starting from TiDB v5.4.0, TiKV introduces the auto-tune feature for backup tasks. For clusters in v5.4.0 or later versions, this feature is enabled by default. When the cluster workload is heavy, the feature limits the resources used by backup tasks to reduce the impact on the online cluster. For more information, refer to [BR Auto-Tune](/br/br-auto-tune.md).
+
+TiKV supports [dynamically configuring](tikv-control.md#modify-the-tikv-configuration-dynamically) the auto-tune feature. You can enable or disable the feature by the following methods without restarting your cluster:
```
Comment (en-jin19) [inline br/backup-and-restore-faq.md:16]:
```suggestion
TiKV supports [dynamically configuring](/tikv-control.md#modify-the-tikv-configuration-dynamically) the auto-tune feature. You can enable or disable the feature by the following methods without restarting your cluster:
```

File: br/backup-and-restore-faq.md (hunk 15 above)
Comment (ran-huang) [reply br/backup-and-restore-faq.md:44]: 1. master 和 5.4 里不需要有 5.3 啊。每个版本的文档只需要有该版本的信息。
2. OK

Comment (en-jin19) [reply br/backup-and-restore-faq.md:44]: 之前想一起呈现的话可能会有对比效果，不过如果一个版本中建议只呈现当前版本信息话，那我再提个中文 PR 把 cp 到 master 和 5.4 的中文文档内容修改一下吧，@ran-huang @YuJuncen 两位看如何？

Comment (YuJuncen) [reply br/backup-and-restore-faq.md:44]: 我感觉 5.4 不管留不留 5.3 的信息我都能接受吧 🤔

File: br/backup-and-restore-faq.md [hunk 19]
```diff
@@ -16,9 +32,17 @@ It is recommended to mount an NFS disk as a backup disk during backup. For detai
 
 ## How much does it affect the cluster during backup using BR?
 
-When you use the `oltp_read_only` scenario of `sysbench` to back up to a disk (make sure the backup disk and the service disk are different) at full rate, the cluster QPS is decreased by 15%-25%. The impact on the cluster depends on the table schema.
+For TiDB v5.4.0 or later versions, BR not only reduces the default CPU utilization used by backup tasks but also introduces the auto-tune feature. When this feature is enabled, BR automatically limits the resources used by backup tasks when performing backups in the cluster with high workloads, thereby limiting the speed of BR backups. Therefore, when using the default configuration for backup tasks in the cluster in v5.4.0 with high workloads, the impact of the tasks on the cluster performance is significantly less than the impact for the clusters earlier than v5.4.0. For details on the auto-tune feature, see [BR Auto-tune](/br/br-auto-tune.md).
```
Comment (ran-huang) [inline br/backup-and-restore-faq.md:35]:
```suggestion
For TiDB v5.4.0 or later versions, BR not only reduces the default CPU utilization used by backup tasks but also limits the resources used by backup tasks in the cluster with heavy workloads. Therefore, when you use the default configuration for backup tasks in the v5.4.0 cluster with heavy workloads, the impact of the tasks on the cluster performance is significantly less than the impact for the clusters earlier than v5.4.0. For details, see [BR Auto-tune](/br/br-auto-tune.md).
```

File: br/backup-and-restore-faq.md [hunk 20]
```diff
@@ -16,9 +32,17 @@ It is recommended to mount an NFS disk as a backup disk during backup. For detai
...
+
+The following is an internal test on a single node. The test results show that when using the default configuration of v5.4.0 and its earlier versions in the **full-speed backup** scenario, the impact of backup using BR on cluster performance is quite different. The detailed test results are as follows:
```
Comment (ran-huang) [inline br/backup-and-restore-faq.md:37]:
```suggestion
The following is an internal test on a single node. The test results show that when you use the default configuration of v5.4.0 and v5.3.0 in the **full-speed backup** scenario, the impact of backup using BR on cluster performance is quite different. The detailed test results are as follows:
```

File: br/backup-and-restore-faq.md [hunk 21]
```diff
@@ -16,9 +32,17 @@ It is recommended to mount an NFS disk as a backup disk during backup. For detai
...
+
+If you need to manually control the impact of backup tasks on cluster performance, you can use the following solutions. These two methods can reduce the impact of backup tasks on the cluster, but they also reduce the speed of backup tasks.
 
-To reduce the impact on the cluster, you can use the `--ratelimit` parameter to limit the backup rate.
+- Use the `--ratelimit` parameter to limit the speed of backup tasks. Note that this parameter limits the speed of **saving backup files to external storage**. When calculating the total size of backup files, use the `backup data size(after compressed)` in the backup log as a benchmark.
+- Adjust the TiKV configuration item [`backup.num-threads`](/tikv-configuration-file.md#num-threads-1) to limit the resources used by backup tasks. This configuration item determines the number of threads used by backup tasks. The test data shows that when the number of threads used by backup tasks using BR does not exceed `8`, and the total CPU utilization of the cluster does not exceed 60%, the backup tasks have little impact on the cluster, regardless of the read and write load.
```
Comment (ran-huang) [inline br/backup-and-restore-faq.md:45]:
```suggestion
- Adjust the TiKV configuration item [`backup.num-threads`](/tikv-configuration-file.md#num-threads-1) to limit the resources used by backup tasks. This configuration item determines the number of threads used by backup tasks. When BR uses no more than `8` threads for backup tasks, and the total CPU utilization of the cluster does not exceed 60%, the backup tasks have little impact on the cluster, regardless of the read and write workload.
```

**Review by ran-huang — APPROVED** [review]

File: br/backup-and-restore-faq.md [hunk 22]
```diff
@@ -8,6 +8,22 @@ aliases: ['/docs/dev/br/backup-and-restore-faq/']
 
 This document lists the frequently asked questions (FAQs) and the solutions about Backup & Restore (BR).
 
+
```
Comment (TomShawn) [inline br/backup-and-restore-faq.md:11]:
```suggestion
```

**Review by YuJuncen — APPROVED** [review]

**Pull request merged**